visualizer/examples.js

```diff
@@ -162,6 +226,12 @@
         setExample(selectedId, cm.getValue());
         saveExamples();
       }
+    },
+    'Alt-S': function(cm) { // save (windows)
```

Comment:
Shouldn't it be `'Ctrl-S'`? And I would factor this function out so that both shortcuts can use the same implementation.